src/code/z_effect_soft_sprite.c

```diff
@@ -242,11 +247,12 @@ void EffectSs_Spawn(PlayState* play, s32 type, s32 priority, void* initParams) {
 
     if (profile->init(play, index, &sEffectSsInfo.table[index], initParams) == 0) {
         PRINTF(VT_FGCOL(GREEN));
-        // "Construction failed for some reason. The constructor returned an error.
-        // Ceasing effect addition."
-        PRINTF("EffectSoftSprite2_makeEffect():"
-               "何らかの理由でコンストラクト失敗。コンストラクターがエラーを返しました。エフェクトの追加を中"
-               "止します。\n");
+        PRINTF(T("EffectSoftSprite2_makeEffect():"
+                 "何らかの理由でコンストラクト失敗。コンストラクターがエラーを返しました。"
+                 "エフェクトの追加を中止します。\n",
+                 "EffectSoftSprite2_makeEffect(): "
+                 "Construction failed for some reason. The constructor returned an error. "
```

Comment:
didnt notice till now but some of these translations end with a space, is this an artifact of the script or this particular case?

Reply:
This is just a long line broken over several source lines but without an actual line return \n inside, there are spaces after punctuation like usual English